lib/libutee/include/pta_system.h

```diff
@@ -182,4 +182,13 @@
  */
 #define PTA_SYSTEM_DLSYM                11
 
+/*
+ * Retrieves a copy of the TPM Event log held in secure memory.
+ *
+ * [out]	memref[0]: Pointer to a buffer where to store the event log.
+ *			   Upon returning, size will be updated with the size
+ *			   of the actual log.
```

Comment:
"Upon returning..." is not useful, since this is the way an [out] memref parameter is expected to work always.